config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```diff
@@ -193,23 +161,23 @@ periodics:
           memory: "8Gi"
 
 # manual-release-bump-required
-- interval: 2h
+- interval: 1h
   name: ci-kubernetes-build-stable3
   labels:
     preset-service-account: "true"
     preset-dind-enabled: "true"
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/bootstrap:v20190211-fab6034b4
+    - image: gcr.io/k8s-testimages/bootstrap:v20190213-2f4f06285
       args:
-      - --repo=k8s.io/kubernetes=release-1.10
+      - --repo=k8s.io/kubernetes=release-1.11
       - --repo=k8s.io/release
       - --root=/go/src
       - --timeout=180
       - --scenario=kubernetes_build
       - --
       - --allow-dup
-      - --extra-publish-file=k8s-stable3
+      - --extra-publish-file=k8s-stable2
```

Comment:
same here, job name is `stable3` vs the use of `stable2` here, and an interval bump I don't understand

Reply:
done